oda_api/api.py

```diff
@@ -1317,6 +1335,33 @@ def set_api_code(query_dict, url="www.astro.unige.ch/mmoda/dispatch-data"):
 
         return _cmd_
 
+
+    def save_result(self, response_json):
+        fn = self.unique_response_json_fn
+
+        os.makedirs(os.path.dirname(fn), exist_ok=True)
+
+        json.dump(response_json, gzip.open(fn, "wt"))
```

Comment:
Generally, I always prefer using `with` when opening file

Reply:
yeah, well, it adds an extra line and an extra variable. Ultimately the file would get closed.
But ok, it's safer to make sure it's closed here.